cobigen/cobigen-core-parent/cobigen-core-test/src/main/java/com/devonfw/cobigen/test/assertj/GenerationReportToAssert.java

```diff
@@ -43,4 +43,43 @@ public GenerationReportToAssert isSuccessful() {
         }
         return this;
     }
+
+    /**
+     * Checks if an Exception of the given class was found in the {@link GenerationReportTo} error reports.
+     *
+     * @param exception
+     *            Class of Exception to check for
+     * @return {@link GenerationReportToAssert}
+     * @throws AssertionError
+     *             if another or no Exception was found in the report
+     */
+    public GenerationReportToAssert containsException(Class exception) throws AssertionError {
```

Comment:
I think `Class` will give a compiler warning, please make it `Class<? extends Throwable>`

Reply:
adjusted.